### PR TITLE
Alewis/delete tail

### LINKED ISSUE #1245: send DELETE /tail on shutdown

### PR DESCRIPTION
**Pull request by ashleymichal**
closes #1245 

- [x] merge https://github.com/cloudflare/cloudflare-rs/pull/119
- [x] update cloudflare-rs to next version

### CODE REVIEW

File: src/tail/mod.rs [hunk 1]
```diff
@@ -21,7 +21,6 @@ use shutdown::ShutdownHandler;
 use tunnel::Tunnel;
 
 use console::style;
-use tokio;
```
Comment (ashleymichal) [inline src/tail/mod.rs:24]: clippy 💅

File: src/tail/session.rs [hunk 2]
```diff
@@ -3,11 +3,10 @@ use std::time::Duration;
 
 use indicatif::{ProgressBar, ProgressStyle};
 use regex::Regex;
-use reqwest;
```
Comment (ashleymichal) [inline src/tail/session.rs:6]: clippy 💅

File: src/tail/session.rs [hunk 3]
```diff
@@ -32,16 +33,36 @@ impl Session {
         metrics_port: u16,
         verbose: bool,
     ) -> Result<(), failure::Error> {
+        let mut session = Session { tail_id: None };
```
Comment (EverlastingBugstopper) [inline src/tail/session.rs:36]: why no tail id

Comment (ashleymichal) [reply src/tail/session.rs:36]: it isn't populated yet at this point; in `start`, we call the api to initiate the session, then populate this with the response. should i leave an inline comment?

Comment (EverlastingBugstopper) [reply src/tail/session.rs:36]: i'd just make it `Session::new()` tbh

**Pull request merged**